app/models/video.py

```diff
@@ -0,0 +1,20 @@
+from flask import current_app
+from app import db
+from sqlalchemy.orm import relationship
+from sqlalchemy.ext.declarative import declarative_base
+
+class Video(db.Model):
+    video_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime)
+    total_inventory = db.Column(db.Integer, default=0)
+    available_inventory = db.Column(db.Integer, default=total_inventory)
```

Comment:
As an aside, I don't think setting the default to total_inventory will work - it never gets called, because the only time a new video is created (routes.py, line 91) a value is passed in for available_inventory, so there is never an attempt to use the total_inventory as a default.